README.md

```diff
@@ -1,2 +1,38 @@
 # hw01
 Repository for homework 1
+------------------------------
+# A bio for David Whyman!
+ 
+*David Whyman was born in Washington, DC on June 5, 1998.*
+
+### David's Education
+David graduated from Walt Whitman High School in Bethesda Maryland. Now, David is a public policy major at the University of Chicago.
+
+### David's Hobbies!
+* Travel
+* Literature
+  * Magical Realism
+* Art
+   * Figure Drawing
+   * Landscape Painting
+* Horseback Riding 
+
+### About David
+I look at the world in an analytical way. Deconstructing complex issues and seeing how all the little parts interact is really cool. That's why I'm curious about social science research. Computer science and statistics provide useful tools to analyze these issues. Oh and by the way I love dogs. Especially **golden retrievers**.
+
+### David's Activities
+At the University of Chicago, I am involved in the Paul Douglas Institute, *The Citizen Bulletin*, and *The Maroon*. Check out [this article](https://www.chicagomaroon.com/article/2018/3/30/preckwinkle-talks-soda-tax-criminal-justice/) I wrote for *The Maroon*.
+
+
+
+<img style="float:left;transform: rotate(0deg); width:300px" src="https://github.com/whymand/myrepo/blob/master/Biopic.jpg" />
+
```

Comment:
This is a bit of an odd addition here, and it sounds like you had a little bit of an issue getting the picture to insert.  One way to do it is save it on your computer under the folder that your homework project is in.  Then you include it almost just like you would with a link, but just putting the name of the document in the ().  So, for example, if you save the image as "Biopic.jpg" in the correct folder, then you can link it as:

[](Biopic.jpg)

When you commit and push to Git, the image will also be uploaded and will show up on your R markdown.